src/app/order/OrderStatus.tsx

```diff
@@ -18,7 +18,15 @@ const OrderStatus: FunctionComponent<OrderStatusProps> = ({
 }) => {
 
     const getMandateProvider = useCallback(() => {
-        return order?.payments?.[0].description === 'Stripe (SEPA)' ? 'SEPA Direct Debit' : order?.payments?.[0].description;
+        const Mandates = [
+            { method: 'Stripe (SEPA)', value: 'SEPA Direct Debit Mandate' },
```

Comment:
should all these strings be translatable?

Reply:
It is being translated on line 51, as these string are used in a place holder of order_confirmation.mandate_link_text

Reply:
Changed it, in a way similar as you proposed